Package.swift

```diff
@@ -0,0 +1,24 @@
+// swift-tools-version:5.1
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "URBNAlert",
+    platforms: [ .iOS(.v10)],
+    products: [
+        // Products define the executables and libraries produced by a package, and make them visible to other packages.
+        .library(
+            name: "URBNAlert",
+            targets: ["URBNAlert"]),
+    ],
+    dependencies: [
+    .package(url: "https://github.com/urbn/URBNSwiftyConvenience.git", from: Version(1, 7, 0))
```

Comment:
I think (reading the [documentation](https://github.com/apple/swift-package-manager/blob/master/Documentation/PackageDescription.md#methods-3) that we just need `from: "1.7.0"` and it’ll infer this.

Also, it might be easier to indent this.

Reply:
@ncsouvenir ☝️

Reply:
@codecaffeine  - gotcha. Updated that argument